hvpy/api_groups/screenshots/get_tile.py

```diff
@@ -0,0 +1,68 @@
+from typing import Optional
+from datetime import datetime
+
+from pydantic import validator
+
+from hvpy.io import HvpyParameters, OutputType
+from hvpy.utils import convert_date_to_isoformat
+
+
+class getTileInputParameters(HvpyParameters):
+    """
+    Handles the input parameters of the ``getTile`` API.
+
+    Attributes
+    ----------
+    {Shared}
+    id
+        Unique image identifier.
+    x
+        Tile x-coordinate.
+    y
+        Tile y-coordinate.
+    imageScale
+        Image scale in arcseconds per pixel.
+    difference
+        Specify image type difference.
+        0 - Display regular image
+        1 - Running difference image
+        2 - Base difference image
```

Comment:
We need to check that this renders correctly in the documentation.

Reply:
No its all collapsed: https://hvpy--59.org.readthedocs.build/en/59/api/hvpy.getTile.html

This needs to be changed.